### ISSUE TITLE
Replace cache pull to cache fetch 

### ISSUE THREAD

**Issue by dmpetrov** [dataversioncontrol/dvc#521]
Why don't we use Git approach:

`dvc cache fetch` instead of `dvc cache pull`
`dvc cache pull` --> `dvc cache fetch && dvc checkout`

**Comment by dmpetrov** [comment]
I realized that `dvc cache pull` is already working as `dvc cache fetch && dvc checkout`
So, we should think about introducing `dvc cache fetch` without checking out.

**Comment by efiop** [comment]
I totally agree. I'll introduce 'dvc cache fetch' soon, it is a quick addon after all.
